### PR TITLE
Fix/Uar 431 ceased date field and validation roe api - error message fixed

### PR DESCRIPTION
**Pull request by cdrainCH**
### JIRA link

https://companieshouse.atlassian.net/jira/software/c/projects/UAR/boards/481?modal=detail&selectedIssue=UAR-431&sprint=2781

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by corybudd-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public static boolean isDateWithinLast3Months(LocalDate compareToDate, String qu
     public static boolean isCeasedDateAfterStartDate(LocalDate ceasedDate, LocalDate startDate, String qualifiedFieldName, Errors errors, String loggingContext) {
         if (ceasedDate.isBefore(startDate) || ceasedDate.isEqual(startDate)) {
             setErrorMsgToLocation(errors, qualifiedFieldName, ValidationMessages.CEASED_DATE_BEFORE_START_DATE_ERROR_MESSAGE.replace("%s", qualifiedFieldName));
-            ApiLogger.infoContext(loggingContext, qualifiedFieldName + " ceased date should be after start date");
+            ApiLogger.infoContext(loggingContext, qualifiedFieldName + " must be after the appointed date");
```
Comment (stephanie-ch) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java:38]: use the ValidationMessages.CEASED_DATE_BEFORE_START_DATE_ERROR_MESSAGE variable here?

Comment (cdrainCH) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java]: Fixed

**Review by stephanie-ch — APPROVED** [review]

**Comment by stephanie-ch** [comment]
@cdrainCH Please reply to the CH comments on your previous PR with this PR

**Review by AndreKramer7654 — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]
Thanks for making that change and looks good to me. Also agree that method name is fine.

**Pull request merged**